packages/react/src/river/RiverAccordion/RiverAccordion.module.css

```diff
@@ -0,0 +1,161 @@
+.RiverAccordion {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto;
+  grid-template-areas:
+    'visual'
+    'accordion';
+  gap: 0;
+  position: relative;
+  interpolate-size: allow-keywords;
```

Comment:
This isn't supported in Safari or baseline, can you do it another way?

Reply:
Let me look into it 👍

Reply:
Replaced `interpolate-size: allow-keywords;` with a `grid-template-rows` transition, as we do on the homepage.